affiliates/mexico.swagger.json

```diff
@@ -420,11 +420,13 @@
           },
           "employed_since" : {
             "type" : "string",
-            "example" : "yyyy-mm-dd"
+            "example" : "yyyy-mm-dd",
```

Comment:
I doubt this example will pass as a value

Reply:
This could be added as description

Reply:
fixed date examples